### ISSUE TITLE
Feature request: support secure HTTPS local dev environnement

### ISSUE THREAD

**Issue by borisschapira** [netlify/netlify-dev-plugin#137]
Right now, I'm developping locally with a compiled Jekyll with a self-signe cert. To use Netlify Dev, I would need it to support secure environment and the ability to either produce it's own self-signed cert, or be able to pass mine as argument.

**Comment by calavera** [comment]
I think generating the certificates might be more intuitive, and it will require less friction. We can add an option like `--secure` and make everything happen. That should also work with the live tunnel `--live --secure`, although the tunnel connection is already secure by default.

**Comment by Booligoosh** [comment]
Yes please!!! For this reason alone at the moment I can't yet use Netlify Dev, as soon as it's added I'll be able to switch my whole workflow over. Thanks :)
